src/FixedDataTable.js

```diff
@@ -630,9 +630,16 @@ class FixedDataTable extends React.Component {
 
     const {
       className,
+      columnOffsets,
```

Comment:
All the offsets are currently calculated by columnWidth selector, and not the reducer.